frontstage/views/surveys/help/surveys_help.py

```diff
@@ -393,6 +290,58 @@ def send_help_message(session, survey_ref, ru_ref):
         return redirect(url_for("surveys_bp.get_survey_list", tag="todo"))
 
 
+def abort_help_if_session_not_set():
+    """
+    Aborts the process if flask session attributes are not set
+    """
+    if flask_session["help_survey_ref"] is None or flask_session["help_ru_ref"] is None:
+        abort(404)
```

Comment:
Can you add a log.info line saying `both keys not in session, aborting` or something like that?